Official Savage Worlds/SavageWorldsCharSheet.html

```diff
@@ -3271,6 +3283,12 @@
             typeof lookupObj.pace !== 'undefined'
               ? (finalCharacterDetails['pace'] = lookupObj.pace)
               : log('Pace', 'Not found', r20color);
+            // set pace mod
+            // set parry mod
+            // set toughness mod
+            ['pace','parry','tufness'].forEach(attr => {
```

Comment:
should this be `'toughness'`?

Reply:
Heh, no. It's apparently a misspell from back when the sheet was first made. Fixing it now would require changing too much other code.

Reply:
confirmed this is intended via slack